support-frontend/assets/pages/[countryGroupId]/checkout/components/PersonalDetailsFields.tsx

```diff
@@ -66,6 +77,42 @@ export function PersonalDetailsFields({
 					}}
 				/>
 			</div>
+			{requireConfirmedEmail && !isEmailAddressReadOnly && (
+				<div ref={confirmEmailRef}>
```

Comment:
We can't put the ref on the source component see this [discussion](https://github.com/guardian/source/pull/1106#issuecomment-951761234)